api/v1beta2/helmchart_types.go

```diff
@@ -79,6 +79,11 @@ type HelmChartSpec struct {
 	// +deprecated
 	ValuesFile string `json:"valuesFile,omitempty"`
 
+	// IgnoreMissingValuesFiles controls whether to silently ignore missing values
+	// files rather than failing.
+	// +optional
+	IgnoreMissingValuesFiles bool `json:"ignoreMissingValuesFiles,omitempty"`
```

Comment:
Please add a new section to the API docs and document this field behaviour, after https://github.com/fluxcd/source-controller/blob/main/docs/spec/v1beta2/helmcharts.md#values-files

Reply:
Thank you. Done.